main.go

```diff
@@ -430,7 +430,25 @@ func installOption(listAll bool, custBinPath, mirrorURL *string) {
 
 // install when tf file is provided
 func installTFProvidedModule(dir string, custBinPath, mirrorURL *string) {
-	logger.Infof("Reading required version from terraform module at %q", dir)
+	curDir, err := os.Getwd()
```

Comment:
`tfswitch` reads TF version constraints from several different places. Logging this activity helps user to understand why specific version of TF was installed rather than what user expected.
The code around cur/abs/rel paths in this function is to provide more details on which TF root module is used to read in TF version constraints (e.g. when `-c, --chdir=value` arg is provided). Currently it outputs absolute path which may seem excessive, e.g.:
```shell
> terraform-switcher
17:35:58.206 INFO [main.go:371,getParamsTOML] Reading ".tfswitch.toml" configuration from "home directory"
17:35:58.216 INFO [main.go:436,installTFProvidedModule] Reading required version from terraform module at "/home/giermulnik/data/projects/github.com/some_org/some_repo/path/to/directory/with/root/module"
```
(the path is obfuscated, but in reality it is even longer in my use case)
With this change it would output `[…] Reading required version from terraform module at "."`, which is a way more concise. And with `-c` it outputs relative path for convinience:
```shell
terraform-switcher -c /home/giermulnik/data/projects/github.com/some_org/some_repo/path/to/directory2/with/another/module
15:49:42.505 INFO [main.go:371,getParamsTOML] Reading ".tfswitch.toml" configuration from "home directory"
15:49:42.526 INFO [main.go:448,installTFProvidedModule] Reading required version from terraform module at "../../../directory2/with/another/module"
```

If this makes sense, I'll add an explanatory comment to the code then. Let me know.

Reply:
Let me test this out a bit more before merging

Reply:
@warrensbox A kind reminder.